src/commands/exit.py

```diff
@@ -4,12 +4,12 @@
 from src.shell import Shell
 from src.utils.io import Io
 
-class Exit:
 
+class Exit:
     @staticmethod
     def do(shell: Shell, line: str):
         Io.print("You're quitting cryptext")
-        shell.exit(0)
+        shell.close()
```

Comment:
Here the close() renaming is simply to stick more to conventions, nothing structural